docs/network-iot/run-an-lns/fund-an-oui.mdx

```diff
@@ -0,0 +1,94 @@
+---
+id: fund-an-oui
+title: Fund An Organizationally Unique Identifier
+pagination_label: Fund An Organizationally Unique Identifier
+sidebar_label: Fund an Organizationally Unique Identifier
+description: Helium Documentation
+image: https://docs.helium.com/img/link-image.png
+slug: /iot/run-an-lns/fund-an-oui
+---
+
+[Data Credits](/tokens/data-credit) are used to pay for all transmission of data from devices on the
+Helium Network.
+
+It is worth noting that HNT to DC conversion is irreversible. Data Credits cannot be refunded,
+exchanged for other tokens, or transferred between wallets.
+
+Several options are available for funding an OUI with Data Credits:
+
+- [Fund an OUI using HNT with the Helium Wallet App](fund-an-oui-using-hnt)
+- [Fund an OUI using a credit card with the Sphere Data Credit Portal](/tokens/data-credit-portal)
+
+## Estimating Usage
+
+<figure className="screensnippet-wrapper">
+  <iframe
+    src="https://dune.com/embeds/2608923/4325514"
+    frameborder="0"
+    width="100%"
+    height="350px"
+  ></iframe>
+  <figcaption>
+    Up-to-date usage metrics for all OUIs on the Helium network. <br /> To get a sense of traffic
+    and DC usage on the network, check out the full{' '}
+    <a href="https://dune.com/helium-foundation/helium-data-credits">Dune Data Credit Dashboard</a>.
+  </figcaption>
+</figure>
+<br />
+
+Data credits are charged per 1DC per 24b increment for each response of an uplink payload. For a
+helpful visualization of DC usage of devices for a given period, check out the
+[Data Credit calculator](/tokens/data-credit#data-transfer-calculator) on the DC documentation.
+
+Only uplinks are charged on Helium. Downlinks and join requests are not billed.
+
+### Understanding Multibuy (`max_copies`)
```

Comment:
This core content probably belongs elsewhere with some notes here around the impacts on DC balance when leveraging multibuy.